### ISSUE TITLE
Searching for RSS feed with query string results in 403 Forbidden error

### ISSUE THREAD

**Issue by Art4** [friendica/friendica#14520]
- [x] I have searched open and closed issues for duplicates

Maybe refs #14303.

### Bug Description

I want to add a RSS feed as a contact, so I paste this URL into the contact search:

https://www.youtube.com/feeds/videos.xml?channel_id=UC1t9VFj-O6YUDPQxaVg-NkQ

But friendica responses with a 403 Forbidden error.

I tried different things and explored that the `?` for the query string is the reason for this error:

- search for `https://www.youtube.com/feeds/videos.xml` shows a "no results" error (as expected)
- search for `https://www.youtube.com/feeds/videos.xml?` (note the `?`) results in the 403 Forbidden error

### Steps to Reproduce

1.  Paste `https://www.youtube.com/feeds/videos.xml?` into the contact search
2. Click on "Search" or press [Enter]

Actual Result:

3. I see a white page with the error "Forbidden You don't have permission to access this resource."

Expected Result:

3. I should see the friendica website with a "no results" error 

### Screenshots

actual result:

![grafik](https://github.com/user-attachments/assets/0f8e7bfc-2783-4033-96c6-0f1d492bd554)

Expected result:

![grafik](https://github.com/user-attachments/assets/6b9ae3f7-1322-4ac2-b24d-3d73ab10b3f0)

### Platform Info

Friendica Version: Friendica 'Yellow Archangel' 2024.08 - 1571

Database version: 1571/1571 
Post-Update version: 1550/1550.

Friendica Source: https://github.com/friendica/friendica/tree/2024.08

PHP version: 8.3.12

SQL version:

**Comment by annando** [comment]
Yes, this is caused by the situation described in https://github.com/friendica/friendica/issues/14303.
